### PR TITLE
Add dotnet containers to L3

### PR DESCRIPTION
**Pull request by grisu48**
All dotnet containers should container the L3 label. Add them  to the L3_CONTAINER definition list.

### CODE REVIEW

**Review by dcermak — APPROVED** [review]

**Comment by grisu48** [comment]
Metadata failures are expected, because the released containers don't have the new label yet.

**Review by dcermak — CHANGES_REQUESTED** [review]
Blocking this until we have the final call from PM

**Comment by grisu48** [comment]
Following the feedback from Michal this PR is obsolete.

**Pull request closed without merging**
